src/systemd/sd-dhcp-server.h

```diff
@@ -40,6 +44,10 @@ int sd_dhcp_server_attach_event(sd_dhcp_server *client, sd_event *event, int64_t
 int sd_dhcp_server_detach_event(sd_dhcp_server *client);
 sd_event *sd_dhcp_server_get_event(sd_dhcp_server *client);
 
+typedef void (*sd_dhcp_server_callback_t)(sd_dhcp_server *server, int event, void *userdata);
```

Comment:
So let's make this uint64_t to have the same width everywhere.

Reply:
ok